app/javascript/gobierto_visualizations/modules/subsidies_controller.js

```diff
@@ -118,6 +118,12 @@ export class SubsidiesController {
   setGlobalVariables(rawData) {
     let subsidiesData = rawData[0];
 
+    subsidiesData.sort((a, b) => {
+      a = a.grant_date.replace(/-/g, '');
+      b = b.grant_date.replace(/-/g, '');
+      return a.localeCompare(b)
+    });
```

Comment:
What are you doing with the replace clauses exactly? Have you tried if any localCompare method _options_ works to you? Check the [available options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator).

On the other hand, what if is there no `grant_date`? 💥 what if `grant_date` is not an string? 💥

Reply:
Done!